packages/ui/src/composites/wui-list-account/index.ts

```diff
@@ -109,7 +112,7 @@ export class WuiListAccount extends LitElement {
     let label = this.labels?.get(this.accountAddress)
 
     if (!label && this.connectedConnector === 'AUTH') {
-      label = `${this.accountType === 'eoa' ? 'Email' : 'Smart'} Account`
+      label = `${this.accountType === 'eoa' ? this.socialProvider ?? 'Email' : 'Smart'} Account`
```

Comment:
Is `this.socialProvider` have capitalized text (since we use it for icon name as well, wanted to double check)? If not should we use `text-transform: capitalize` in CSS side?